### ISSUE TITLE
Icons are disappearing

### ISSUE THREAD

**Issue by BENZIN69** [mantisbt-plugins/BBCodePlus#106]
Hello dear !

thank you very much for your plugin it is very very useful !

there is something weird when i'm using it with an other plugin : To-do lists 2.5.0

as soon as I activate the Todolist plugin, the BBCOde+ plugin icons disappear

![160082589-45df09e5-a1d9-4dfc-b69d-82aea7ad3116](https://user-images.githubusercontent.com/102351786/161909092-4c47d0c7-5200-460e-a937-881a1eb0c978.png)
![160082767-6522c17f-a91f-4fdd-8590-3a350448fc1d](https://user-images.githubusercontent.com/102351786/161909108-742dd5d3-f2b6-4228-8e56-96f0859ec8c3.png)

i'm working on mantis 2.25.2

Maybe you will be able to fix something.

Thank you again for your work !

David

**Comment by bkraul** [comment]
Check the developer tools console (in your browser, F12). More than likely you have a CSP error.

**Comment by bkraul** [comment]
You can refer to [this issue](https://github.com/mantisbt-plugins/BBCodePlus/issues/3)

**Comment by TomR** [comment]
Had the same problem.
Maybe due to the fact that there is somewhere a newline or space after a PHP end tag.

When you look at view source, at what line is source starting with code?
It should be line 1, if it start lower, then that is de cause of your problem.

**Comment by chkao-ui** [comment]
Is there a solution to the problem shown in the picture?

I still have the icon disappearance in 2.1.19.
![1717743819397](https://github.com/mantisbt-plugins/BBCodePlus/assets/172014596/235c4a19-d16a-4736-8c95-8b7738d70573)

MantisBT Ver 2.25.4

**Comment by Albusoline** [comment]
Got a similar issue:
Copied whole mantis from one place to another, everything works well except the icons of the BBPlugin.
In the old place, icons are showing up, in the new place, there are gone (looking pretty much the same as screenshot above)
Absolutely identical installation.
![grafik](https://github.com/user-attachments/assets/dbc655af-1f12-47ec-955e-8334fba8458a)

Important:
It is on the same webspace, just one folder further down in the structure:

Old:
/issues/

new:
/new/issues/

Mantis: 2.25.7
php: 7.4.33
BBCode+ 2.1.19

**Comment by bkraul** [comment]
What does the Developer Tools console say? If you notice, that's mentioned earlier in this thread.

**Comment by Albusoline** [comment]
Thanks for your comment. I didn't see it initially.

Unfortunately I'm too late to get any information from the developer console. It seems that something happened on the server side during the night, because this morning it was running without any issues despite no changes on my side. Hmm....

The one and only change was a reboot of my computer. But because the error also happened on another computer of my collegue, I don't think it is related to my personal settings (Cache, Cookies whatever) on my computer.

**Comment by bkraul** [comment]
No, if you read the thread you'll see that it refers to CSP. That's a server-side situation. But that's the sort of stuff you would see on the developer tools console.
